src/pages/HomePage/index.js

```diff
@@ -50,55 +38,53 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const HomePage = (props) => {
-  const {
-    home = [],
-    common: { init = false, error = null },
-  } = props;
-
+  const { home = [] } = props;
   const classes = useStyles();
+  useEffect(() => {
+    props.fetchFirebaseData();
```

Comment:
Why do you need to fetch firebase data here?

Reply:
You're right, I don't need to use fetchFirebaseData here. Updated in last commit